scripts/hostcfgd

```diff
@@ -1240,9 +1240,12 @@ class SshServer(object):
                 syslog.syslog(syslog.LOG_ERR, "Ssh {} {} out of range".format('port', port_num))
                 return False
         port_line_num = self.get_line_num_of_pattern("Port", SSH_CONFG, True)
-        run_cmd("sed -i -E '/^#?Port '[1-9]'/d' " + SSH_CONFG) # remove all lines start with Port or #Port
+        remove_ports_cmd = "sed -i -E '/^#?Port '[1-9]'/d' " + SSH_CONFG # remove all lines start with Port or #Port
+        os.system(remove_ports_cmd)
```

Comment:
Why are we replacing run_cmd with os.system? os.system has security concerns and it is being replaced elsewhere in the code. Please refer to  https://github.com/sonic-net/sonic-utilities/pull/2402

Reply:
Can you please debug further? I feel other commands use run_cmd to do similar things

Reply:
Issue fixed - calls for os.cmd removed. Thanks @dgsudharsan